src/components/Clusters/views/ClusterOverview/ClusterOverview.js

```diff
@@ -64,6 +65,15 @@ export function ClusterOverview() {
               slot="details-top"
               root=""
             />
+            <Title
+              level="H3"
+              style={{
+                ...spacing.sapUiMediumMarginBegin,
+                ...spacing.sapUiMediumMarginTopBottom,
+              }}
+            >
+              {t('cluster-overview.headers.cluster-details')}
+            </Title>
```

Comment:
Why is it moved from `title` prop?

Reply:
the title had to be moved outside of ResourceDetailsCard component in order to position it parallel to ResourceStatusCard, and because ClusterOverview isn't created with ResourceDetails component the title had to be put there separately